CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## Next release
+### Breaking Changes
+- ETH2 Mode - block signing request (BLOCK_V2), starting from BELLATRIX fork, use block_header instead of block. [#547](https://github.com/ConsenSys/web3signer/pull/547)
+
+### Features Added
+- Added support for optimized block signing requests starting from Bellatrix fork. [#437](https://github.com/ConsenSys/web3signer/issues/437)
```

Comment:
Do we get gnosis support with this teku upgrade and do we want to mention that here?

Reply:
I think technically we do get gnosis support, may be we can add that once we pick gnosis ticket?

Reply:
I guess it depends if this is going to be released or not before we pick that up.

Maybe could say "experimental support for gnosis"?

Reply:
Its just that I didn't test gnosis support at all while working on this ticket.

Reply:
that's why I added the experimental...how about "experimental support for gnosis due to teku upgrade"?

Reply:
Once we test it we might not require any code changes and therefore no opportunity to add it to a changelog/release